zebra-network/src/peer_set/set.rs

```diff
@@ -475,6 +484,12 @@ where
                 Some(Ok((key, svc))) => {
                     trace!(?key, "service became ready");
 
+                    if self.bans_receiver.borrow().contains_key(&key.ip()) {
+                        debug!(?key, "service is banned, dropping service");
```

Comment:
```suggestion
                        warn!(?key, "service is banned, dropping service");
```

We don't except bans to happen very often unless there is something really wrong with the peers. Should we increase the log level ?

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/9201/commits/88e0c9c0405ee533f6308394928a3f8de9f161d0.